src/Files.App/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -246,6 +241,12 @@ public async Task LoadDriveIcon()
             }
             Icon = await IconData.ToBitmapAsync();
         }
+
+        private void UpdateHoverDisplayText()
+        {
+            HoverDisplayText = $"{"PropertiesItemPath/Text".GetLocalizedResource()} {(Path.Contains("?", StringComparison.Ordinal) ? Text : Path)}" +
```

Comment:
We should have a generic resource for path, renaming `PropertiesItemPath` to path is probably the best way to do this.

Reply:
Do you mean that it already exists and I should use it, or that I should edit Resources files?

Reply:
I think you can edit the existing resource to just be called `Path`.